packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,153 @@
+# Nimble Accordion
+
+## Overview
+
+The `nimble-accordion` is a vertical stack of interactive headings. The headings are controls that can be used to reveal or hide content in specific sections. The Nimble accordion is based upon [FAST's accordion component](https://github.com/microsoft/fast/tree/master/packages/web-components/fast-foundation/src/accordion).
+
+### Background
+
+[Nimble issue #533: nimble-accordion Component](https://github.com/ni/nimble/issues/533)
+
+[Visual Design](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1295-85131&mode=design&t=DxDRlqT7MeCPLOxi-0)
+
+---
+
+## Design
+
+Simple accordion group
+
+```
+<fast-accordion>
+    <fast-accordion-item>
+        <span slot="heading">Panel one</span>
+        <span slot="icon">^</span>
+        Panel one content
+    </fast-accordion-item>
+    <fast-accordion-item expanded>
+        <span slot="heading">Panel two</span>
+        <span slot="icon">^</span>
+        Panel two content
+    </fast-accordion-item>
+    <fast-accordion-item>
+        <span slot="heading">Panel three</span>
+        <span slot="icon">^</span>
+        Panel three content
+    </fast-accordion-item>
+</fast-accordion>
+```
+
+Complex accordion group with components in slots and single expand-mode
+
+```
+<fast-accordion expand-mode="single">
+    <fast-accordion-item>
+        <fast-checkbox slot="start"></fast-checkbox>
+            <fast-select slot="end">
+                <fast-option value="1">Option 1</fast-option>
+                <fast-option value="2">Option 2</fast-option>
+                <fast-option value="3">Option 3</fast-option>
+            </fast-select>
+                Accordion one content
+            <div slot="heading">Accordion one</div>
+    </fast-accordion-item>
+    <fast-accordion-item>
+        <fast-checkbox slot="start"></fast-checkbox>
+            <fast-number-field placeholder="number" slot="end"></fast-number-field>
+                Accordion two content
+            <div slot="heading">Accordion two</div>
+    </fast-accordion-item>
+    <fast-accordion-item>
+        <fast-checkbox slot="start"></fast-checkbox>
+            <fast-select slot="end">
+                <fast-option value="1">Option 1</fast-option>
+                <fast-option value="2">Option 2</fast-option>
+                <fast-option value="3">Option 3</fast-option>
+            </fast-select>
+            Accordion three content<div slot="heading">Accordion three</div>
+    </fast-accordion-item>
+    <fast-accordion-item>
+        <fast-checkbox slot="start"></fast-checkbox>
+            <fast-number-field placeholder="number" slot="end"></fast-number-field>
+                Accordion four content
+            <div slot="heading">Accordion four</div>
+    </fast-accordion-item>
+</fast-accordion>
+```
+
+### API
```

Comment:
I would organize this section a bit differently. Just start with this list:
-   _Component Name:_
-   _Properties/Attributes:_ 
-   _Methods:_ 
-   _Events:_ 
-   _CSS Classes and Custom Properties that affect the component:_ 
-   _Slots:_

And have a copy of that list for each of the two different components (accoridion and accordion item). Then only go into detail about what deviates from the FAST API (otherwise say "Unchanged"). Much of what you have below is just re-describing the FAST API.